src/lib/components/Treebank/config/config.js

```diff
@@ -1,110 +1,57 @@
-import harrington from './arethusa-configs/harrington.json';
-import jmhgreek from './arethusa-configs/jmhgreek.json';
-import lyonGrec from './arethusa-configs/lyonGrec.json';
-import lyonLatin from './arethusa-configs/lyonLatin.json';
-import midlat from './arethusa-configs/midlat.json';
-import morphgrc from './arethusa-configs/morphgrc.json';
-import morphlat from './arethusa-configs/morphlat.json';
-import pedalion from './arethusa-configs/pedalion.json';
-import persian from './arethusa-configs/persian.json';
-import sanskrit from './arethusa-configs/sanskrit.json';
-import smyth2 from './arethusa-configs/smyth2.json';
-import smyth3 from './arethusa-configs/smyth3.json';
-import snap from './arethusa-configs/snap.json';
-import testlat from './arethusa-configs/testlat.json';
-import turkish from './arethusa-configs/turkish.json';
-import ud from './arethusa-configs/ud.json';
-import vortex from './arethusa-configs/vortex.json';
-import wheelock from './arethusa-configs/wheelock.json';
-import wheelock13 from './arethusa-configs/wheelock1-3.json';
-import wheelock15 from './arethusa-configs/wheelock15.json';
-import wheelock17 from './arethusa-configs/wheelock17.json';
-import wheelock22 from './arethusa-configs/wheelock22.json';
-import wheelock24 from './arethusa-configs/wheelock24.json';
-import wheelock30 from './arethusa-configs/wheelock30.json';
-import wheelock46 from './arethusa-configs/wheelock4-6.json';
-import whitaker from './arethusa-configs/whitaker.json';
-
 const configMap = {
-  'aldt-misc-grc': aldtMiscGrc,
-  'aldt-misc': aldtMisc,
-  'aldt2grc-stage': aldt2grcStage,
-  aldt2grc,
-  'aldt2lat-stage': aldt2latStage,
-  aldt2lat,
-  elmlatch1,
-  elmlatch10,
-  elmlatch10b,
-  elmlatch11,
-  elmlatch2,
-  elmlatch3,
-  elmlatch4,
-  elmlatch5,
-  elmlatch6,
-  elmlatch7,
-  elmlatch9,
-  english,
-  gclat,
-  harrington,
-  jmhgreek,
-  lyonGrec,
-  lyonLatin,
-  midlat,
-  morphgrc,
-  morphlat,
-  pedalion,
-  persian,
-  sanskrit,
-  smyth2,
-  smyth3,
-  snap,
-  testlat,
-  turkish,
-  ud,
-  vortex,
-  wheelock,
-  'wheelock1-3': wheelock13,
-  wheelock15,
-  wheelock17,
-  wheelock22,
-  wheelock24,
-  wheelock30,
-  'wheelock4-6': wheelock46,
-  whitaker,
+  'aldt-misc-grc': true,
+  'aldt-misc': true,
+  'aldt2grc-stage': true,
+  aldt2grc: true,
+  'aldt2lat-stage': true,
+  aldt2lat: true,
+  elmlatch1: true,
+  elmlatch10: true,
+  elmlatch10b: true,
+  elmlatch11: true,
+  elmlatch2: true,
+  elmlatch3: true,
+  elmlatch4: true,
+  elmlatch5: true,
+  elmlatch6: true,
+  elmlatch7: true,
+  elmlatch9: true,
+  english: true,
+  gclat: true,
+  harrington: true,
+  jmhgreek: true,
+  lyonGrec: true,
+  lyonLatin: true,
+  midlat: true,
+  morphgrc: true,
+  morphlat: true,
+  pedalion: true,
+  persian: true,
+  sanskrit: true,
+  smyth2: true,
+  smyth3: true,
+  snap: true,
+  testlat: true,
+  turkish: true,
+  ud: true,
+  vortex: true,
+  wheelock: true,
+  'wheelock1-3': true,
```

Comment:
this will break with this change (probably easier to fix the format file to match the format name, rather than code the ability to deal with this?)

Reply:
I might be wrong, but I think this should still work. The format file's name is [wheelock1-3.json](https://github.com/alpheios-project/arethusa-configs/blob/master/dist/wheelock1-3.json), so if the format string is `wheelock1-3`, it should load that file.